.pre-commit-config.yaml

```
@@ -43,10 +43,10 @@ repos:
       - id: check-yaml
         exclude: ^\.clang-format$
       - id: end-of-file-fixer
-        exclude: ^.*UTF-8-BOM.txt$
+        exclude: ^.*(UTF-8-BOM\.txt|\.ts)$
```

```suggestion
        exclude: ^(res/translations/.*\.ts|.*UTF-8-BOM\.txt)$
```

Otherwise this will also exclude the typescript files in `res/controllers`.